dot/core/helpers_test.go

```diff
@@ -144,7 +167,7 @@ func getGssmrRuntimeCode(t *testing.T) (code []byte) {
 	gssmrGenesis, err := genesis.NewGenesisFromJSONRaw(path)
 	require.NoError(t, err)
 
-	trie, err := genesis.NewTrieFromGenesis(gssmrGenesis)
+	trie, err := wasmer.NewTrieFromGenesis(*gssmrGenesis)
```

Comment:
This is moved to wasmer since it will need to run wasmer code to get the state version from the runtime code contained in the genesis.